main.tf

```diff
@@ -5,11 +5,37 @@ provider "aws" {
 resource "aws_s3_bucket" "demo-s3" {
   bucket = "${var.s3_bucket_prefix}-terraformcloud"
   versioning {
-    enabled = false
-    mfa_delete = false
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket accessible to all authenticated users.**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
S3 bucket Access should be limited to specific users and roles.

source: accurics-